spec/unit/coinbase/wallet_spec.rb

```diff
@@ -34,6 +34,26 @@
       }
     )
   end
+
+  let(:model_with_seed_pending) do
+    Coinbase::Client::Wallet.new(
+      {
+        'id': wallet_id,
+        'network_id': network_id,
+        'server_signer_status': 'pending_seed_creation'
+      }
+    )
+  end
+  let(:model_with_seed_active) do
```

Comment:
These two new `lets` don't seem to be used anywhere?

Reply:
added the tests back